.devcontainer/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ghcr.io/commaai/openpilot-base:latest
+
+RUN printf "\nsource /workspaces/openpilot/tools/openpilot_env.sh\n" >> ~/.bashrc
+# remove gitconfig if exists, since its gonna be replaced by host one
+RUN rm -f /root/.gitconfig
```

Comment:
won't it be shadowed anyway when it's mounted in?

Reply:
We are not mounting it outselves, vscode/devcontainer does that for us, and it fails silently if it exists already.